AmplifyPlugins/Storage/AWSS3StoragePlugin/Operation/AWSS3StorageUploadFileOperation.swift

```diff
@@ -98,6 +98,12 @@ public class AWSS3StorageUploadFileOperation: AmplifyInProcessReportingOperation
             return
         }
 
+        guard FileManager.default.isReadableFile(atPath: request.local.path) else {
```

Comment:
Should we fail earlier ie above the upload size calculation at line 90?

Reply:
Will do